documentation/library_guide/parallel_api/additional_algorithms.rst

```diff
@@ -76,19 +76,28 @@ header.  All algorithms are implemented in the ``oneapi::dpl`` namespace.
     search sequence: [0, 2, 4, 7, 6]
     result sequence: [1, 4, 8, 10, 10]
 
-* ``sort_by_key``: performs a stable key-value sort. The algorithm sorts the sequence's keys according to 
-  a comparioson operator. If no comparator is provided, then the elements are compared with ``operator<``.
-  The sequence's values are permutated according to the sorted sequence's keys. The prerequisite for correct
-  behavior is that the size for both keys sequence and values sequence shall be the same.  
+* ``sort_by_key``: performs a key-value sort. The algorithm sorts a sequence of keys according to
+  a comparison operator. If no comparator is provided, then the elements are compared with ``operator<``.
+  A sequence of values is simultaneously permuted according to the sorted order of keys. For correct behavior,
+  both sequences must be of the same size.

```

Comment:
The given iterator version of `sort_by_key `has 2 iterators for definition of a key sequence and one iterator for definition of a value sequence. 
So, I think the sentence "both sequences must be of the same size" is not quite correct.
For example, we can observe `thrust::sort_by_key ` documentation: 
https://nvidia.github.io/cccl/thrust/api/function_group__sorting_1ga667333ee2e067bb7da3fb1b8ab6d348c.html

Reply:
That's a good catch. Let me think about the wording. 
The documentation of the additional algorithms does not provide the signature of the functions (which is a huge gap on my opinion), what complicates giving a "crisp" description.

Reply:
I've corrected the documentation and improved the tests accordingly.